### ISSUE TITLE
EADL relaxations issue

### ISSUE THREAD

**Issue by FabioCpp** [nrc-cnrc/EGSnrc#277]
In the new release of EGSnrc the "EADL atomic relaxations" is enabled (set to true) by default, on the contrary of what is written in the documentation: "EGSnrc core manual", page 157.

Moreover, with a very simple Montecarlo (e.g. photon of 1 MeV with ecut=0.512 MeV and pcut=0.001 MeV in an infinite region) is possible to demonstrate that the total energy deposited in the material does not correspond to the energy of the starting particle.

In other words, if we set in the AUSGAB routine:  "Etot = Etot + edep"  (with IARG=0,1,2,3,4), the final obtained value for Etot range from 1.000 MeV to about 1.005 MeV when EADL relaxation is enabled, while Etot give the correct value of 1.0000 MeV when EADL is disable (set to .false.).

Is this an expected behaviour?

**Comment by mainegra** [comment]
@FabioCpp the answer is yes, as there is a mismatch between the EADL's and EGSnrc's binding energies. That EADL relaxation is ON by default is a glitch introduced by me in commit #99 where I was actually working on the GUI's and left the $EADL_RELAX macro ON accidentally. This should be reverted back to being OFF until all the issues are fixed. I am currently looking into this as part of the implementation of a more detailed photoelectric interaction model using shellwise photoelectric cross sections.
Having said that what you are observing is most likely the result of slight differences in the binding energies used by the EADL* library compared to the ones used in EGSnrc (Lederer & Shirley 1978). **What was the material you used in the simulations for the 1 MeV photon beam?**
*EPDL97 uses the same binding energies as were used in EPDL89 and EEDL. These are
based on the data of Scofield (J.H. Scofield, Radiative Transitions,in Atomic Inner-Shell Processes, B. Crasemann, ed. (Academic Press, New York, 1975), pp. 265-292.)

**Comment by FabioCpp** [comment]
Thank you for your reply.
I used a CdWO4 compound with 7.9 g/cm3 density, IRAYL=1, ECUT=0.512, PCUT=0.001
But the same spread in the total deposited energy, Etot, can be obtained also when considering an infinite region of lead (i.e. ELEM: PB, IRAYL=1, ECUT=0.512, PCUT=0.001), with Etot ranging between 1.000 MeV and 1.006 MeV, when considering 1 MeV photon beam.

**Comment by ftessier** [comment]
@mainegra this is fully resolved in pull request #282, right?

**Comment by FabioCpp** [comment]
Yes, thank you!

**Comment by FabioCpp** [comment]
Yes, I think so
